app_feup/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/home/reeckset/Programs/flutter"
-export "FLUTTER_APPLICATION_PATH=/home/reeckset/Personal Projects/NIAEFEUP/uni/app_feup"
+export "FLUTTER_ROOT=/opt/flutter"
+export "FLUTTER_APPLICATION_PATH=/home/bdmendes/Documents/NI/app-uni/app_feup"
 export "FLUTTER_TARGET=lib/main.dart"
 export "FLUTTER_BUILD_DIR=build"
 export "SYMROOT=${SOURCE_ROOT}/../build/ios"
-export "FLUTTER_FRAMEWORK_DIR=/home/reeckset/Programs/flutter/bin/cache/artifacts/engine/ios"
+export "FLUTTER_FRAMEWORK_DIR=/opt/flutter/bin/cache/artifacts/engine/ios"
 export "FLUTTER_BUILD_NAME=1.0.0"
-export "FLUTTER_BUILD_NUMBER=1"
+export "FLUTTER_BUILD_NUMBER=3"
```

Comment:
We should add this file to .gitignore and remove its changes from this PR

Reply:
It does seem to be in the gitignore of the ios folder, not sure how it got tracked